modules/virt-querying-metrics.adoc

```diff
@@ -31,6 +31,17 @@ A value above '0' means that the vCPU wants to run, but the host scheduler canno
 ====
 To query the vCPU metric, the `schedstats=enable` kernel argument must first be applied to the `MachineConfig` object. This kernel argument enables scheduler statistics used for debugging and performance tuning and adds a minor additional load to the scheduler.
 ====
+kubevirt_vmi_vcpu_delay_seconds_total::
```

Comment:
You need to add a blank line before you introduce `kubevirt_vmi_vcpu_delay_seconds_total`:: so that the description list renders correctly. See how it looks like regular bolded text in the current rendering? It should match the other query's formatting.

<img width="786" height="703" alt="Screenshot 2025-09-15 at 3 14 36 PM" src="https://github.com/user-attachments/assets/98775ea0-d931-4418-b61f-91845955330b" />

```suggestion
====

kubevirt_vmi_vcpu_delay_seconds_total::
```